lib/remove_bg/upload.rb

```diff
@@ -22,5 +22,9 @@ def self.determine_content_type(file_path)
     end
 
     private_class_method :determine_content_type
+
+    # UploadIO for Faraday < 0.16.0
+    FARADAY_FILE = defined?(Faraday::FilePart) ? Faraday::FilePart : Faraday::UploadIO
```

Comment:
This uses `Faraday::FilePart` where available, as a replacement for the now deprecated `Faraday::UploadIO`:

https://github.com/lostisland/faraday/blob/87ecaf96822ef03ef6e86a9b9b57caa97e4c0fa8/lib/faraday/file_part.rb#L57-L59